cdap-ui/app/features/home/routes.js

```diff
@@ -51,7 +51,20 @@ angular.module(PKG.name+'.feature.home')
           if(!n.length) {
             mySessionStorage.get(PREFKEY)
               .then(function (latest) {
-                var d = latest || rNsList[0].name;
+
+                var def = $filter('filter')(rNsList, {name: 'default'}, true);
+
+                if (def.length < 1) {
```

Comment:
Can we just use === 0? It's not a big deal, but most other places, we do zero comparison, rather than lessThan comparison with 1.